drivers/gles2/rasterizer_scene_gles2.h

```diff
@@ -592,18 +592,18 @@ class RasterizerSceneGLES2 : public RasterizerScene {
 			union {
 				struct {
 					//from least significant to most significant in sort, TODO: should be endian swapped on big endian
-
 					uint64_t geometry_index : 14;
 					uint64_t instancing : 1;
 					uint64_t skeleton : 1;
-					uint64_t shader_index : 10;
+					uint64_t shader_index : 8;
```

Comment:
This limits the number of shaders you can have from 1024 (already dangerously low) to 256. It might be best to figure out a way to do this without adjusting "sort_key". I tried to fiddle with it once before but reduz didn't allow it. He has planned out these values very carefully.